### ISSUE TITLE
Review landing page

### ISSUE THREAD

**Issue by mfarfanr** [mfarfanr/kari#3]
Amy, @amyhayes

Can you review the landing page in your different devices? For the mobile one (smallest breakpoint) I did make a change on the Login and Email links to call-to-action buttons like many websites do. I haven't hooked up the links on the page, but that's a 5-minute change.

The URL is: [http://www.maneli.net/kari/src/project/docs/templates/home.html](http://www.maneli.net/kari/src/project/docs/templates/home.html)

Manolo

**Comment by mfarfanr** [comment]
Manolo,

Not sure if I am to be hitting “reply all” or going to github so I went to the fall back of the email we use with Elizabeth.  Sorry I am not cutting edge with your github tool.
 
I have checked it on a phone, Ipad and desktop It looks very good EXCEPT that the phone numbers on the first 2 show in a blue/purple and are very hard to read.  If I click the numbers I am asked
if I want to CALL… so that is OK but can you change the color to be easier to read?
 
If I click on either version, it jumps me to the old log in page but I assume that is until you finish the task as below. 
 
Let me know when it is ready to go live. 

Thanks,
Amy

**Comment by mfarfanr** [comment]
Amy,

I have made these changes. Phone numbers get those colors natively by the browser, in this case iOS was adding that purple-ish color, whereas Microsoft Edge adds a blue-ish color. Anyway I made a change to avoid those colors and keep the white one.

I also added the proper URLs to the links (including the Kari Corp logo) and removed the top-right viewport size text I was using for debugging purposes.

Let me know if everything looks ok.

Manolo
